state/atoms/trustMetaProfile.ts

```diff
@@ -0,0 +1,9 @@
+import { atom } from 'recoil';
+
+/**
+ * Atom indicating if trustMetaProfile should be set to true
+ */
+export const trustMetaProfile = atom<boolean>({
+  key: 'trustMetaProfile',
```

Comment:
```suggestion
export const trustMetaProfileState = atom<boolean>({
  key: 'trustMetaProfileState',
```
Should both of these be `trustMetaProfileState` to match what we named other atoms?